src/perf/simulations/monitorAnalysis.scala

```diff
@@ -0,0 +1,70 @@
+package default
+
+import java.io.File
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+case class SubmissionStatus(
+	submissionId: String,
+	submissionStatus: String,
+	workflowStatuses: Vector[String]
+)
+
+class monitorAnalysis extends RawlsSimulation {
+
+	val httpProtocol = http
+		.baseURL("https://rawls.dsde-dev.broadinstitute.org")
+		.inferHtmlResources()
+
+	val headers = Map("Authorization" -> s"Bearer ${accessToken}",
+						"Content-Type" -> "application/json") 
+
+	val submissionBody = """{"methodConfigurationNamespace":"alex_methods","methodConfigurationName":"cancer_exome_pipeline_v2","entityType":"pair_set","entityName":"pair_set_1","expression":"this.pairs"}"""
+
+	// var b/c I don't know how to do exec() callbacks
+	var monitorResults: Map[String, SubmissionStatus] = Map()
+
+	val scn = scenario(s"monitorAnalysis_${numUsers}")
+		.feed(tsv("{MAPPING OUTPUT TSV FROM launchAnalysis}")) //feed the workspaceName/submissionId mapping
```

Comment:
can you make it a parameter? i've been adding `val foo = System.getProperty("foo", "maybe_a_default")` in a lot of places so i can script it from bash